pkg/inngest/inngest/_internal/client_lib/client.py

```diff
@@ -28,6 +32,9 @@
 # Dummy value
 _DEV_SERVER_EVENT_KEY = "NO_EVENT_KEY_SET"
 
+MAX_SEND_ATTEMPTS = 5
```

Comment:
renamed this because `MAX_RETRY_ATTEMPTS = 0` would actually mean we don't even try the initial send